Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -124,10 +126,28 @@ def tests
           parallel = false
           ["test/#{test_name}_spec.rb:#{line}"]
         end
+      elsif args.changed?
+        changed_files = Utils.popen_read("git", "diff", "--name-only", "master")
```

Comment:
If this works with `git diff`:

```suggestion
        changed_files = Utils.safe_popen_read("git", "diff", "--name-only", "master")
```

(will abort on non-zero exit codes)

Reply:
Yeah, that makes a lot of sense.

Reply:
Should be fine as long as we don't use `--exit-code`

Reply:
Ah, okay. I'll change it back.